src/sentry/configurations/docs/protocol.md

```diff
@@ -0,0 +1,87 @@
+# Remote Configuration Protocol
+
+Host: https://dsn_url.com # TODO!
```

Comment:
Have we figured out if this will be a new URL, or will we send as a new envelope type?

Reply:
Same as the envelope.  I just haven't looked up its structure yet.